e107_admin/plugin.php

```diff
@@ -665,51 +680,59 @@ private function pluginDownload()
 		$frm = e107::getForm();
 		$mes = e107::getMessage();
 		
+		e107::getDebug()->log("Plugin Download.");
+		$tmp = explode('&amp;', e_QUERY); // #1712 we need to process new ampersand separated query
```

Comment:
I was unsure about leaving debug calls - they only show up in debug mode... when we need them. Agree it adds clutter. Will trim them down. 

Good learning experience. I suppose, why was $_GET and $data are empty may be a better fix ? I'm online today. Busy this evening and weekend.